packages/fx-core/src/component/driver/deploy/azure/azureAppServiceDeployDriver.ts

```diff
@@ -38,8 +39,15 @@ export class AzureAppServiceDeployDriverImpl extends AzureDeployDriver {
     azureResource: AzureResourceInfo,
     azureCredential: TokenCredential
   ): Promise<void> {
+    const startTime = Date.now();
     await this.progressBar?.start();
     await this.zipDeploy(args, azureResource, azureCredential);
     await this.progressBar?.end(true);
+    if (startTime + DeployConstant.DEPLOY_OVER_TIME < Date.now()) {
+      await this.context.logProvider?.info(
+        `Deploying to Azure App Service takes a long time. Consider referring to this document to optimize your deployment: 
```

Comment:
Does this need localization?

Reply:
Agree, this message seems to need localization.